MatrixKit/Views/MXKPieChartHUD.m

```diff
@@ -0,0 +1,93 @@
+//
+//  MXKPieChartHUD.m
+//  Pods
+//
+//  Created by Aram Sargsyan on 8/15/17.
+//
+//
+
+#import "MXKPieChartHUD.h"
+#import "NSBundle+MatrixKit.h"
+#import "MXKPieChartView.h"
+
+@interface MXKPieChartHUD ()
+
+@property (weak, nonatomic) IBOutlet UIView *hudView;
+@property (weak, nonatomic) IBOutlet MXKPieChartView *pieChartView;
+@property (weak, nonatomic) IBOutlet UILabel *titleLabel;
+
+
+@end
+
+@implementation MXKPieChartHUD
+
+#pragma mark - Lifecycle
+
+- (instancetype)initWithFrame:(CGRect)frame
+{
+    self = [super initWithFrame:frame];
+    if (self)
+    {
+        [self configureFromNib];
+    }
+    return self;
+}
+
+- (instancetype)initWithCoder:(NSCoder *)aDecoder
+{
+    self = [super initWithCoder:aDecoder];
+    if (self)
+    {
+        [self configureFromNib];
+    }
+    return self;
+}
+
+- (void)configureFromNib
+{
+    NSBundle *bundle = [NSBundle mxk_bundleForClass:self.class];
+    [bundle loadNibNamed:NSStringFromClass(self.class) owner:self options:nil];
+    self.hudView.frame = self.bounds;
+    
+    self.clipsToBounds = YES;
+    self.layer.cornerRadius = 10.0;
+    
+    self.pieChartView.backgroundColor = [UIColor clearColor];
```

Comment:
You should override [MXKView customizeViewRendering] method in order to move 
the color settings in it.